kdb/key.go

```diff
@@ -71,6 +77,7 @@ func CreateKey(name string, value ...interface{}) (Key, error) {
 	return createKey(name, value...)
 }
 
+// TODO REVIEW: why is this wrapper needed?
```

Comment:
Because this returns the type (*ckey) that allows private functions to use the underlying C pointer without casting from the `Key` interface first

Reply:
Please add such comments as code comments so that later readers of the code also know about this.